src/kubernetes-ingress-controller/guides/using-gateway-api.md

```diff
@@ -13,6 +13,36 @@ Gateway API and Kong's implementation of Gateway API are both in alpha stage and
 under active development. Features and implementation specifics will change
 before their initial general availability release.
 
+{% if_version gte: 2.4.x %}
+## Supported Gateway API Features
+
+Currently, Kong's implementation of Gateway API supports the following resources
+and features:
+
+Supports of `Gateway` and `GatewayClass` resource:
+- Supports one `Gateway` and one `GatewayClass` to be handled by one KIC deployment.
+- Supports `allowedRoutes.namespaces` filters in listeners.
+- Supports referencing of cross-namespace `BackendRefs` if a ReferencePolicy permits them.
+
+Supports of `HTTPRoute` resource:
+- Supports multiple `BackendRefs` with a round-robin load-balancing strategy applied by default across the `Endpoints` or the `Services`, 
+  and also now support weights of `BackendRefs` to enable more fine-tuning of the load-balancing between those backend services.
+- Supports exact, prefix, and regular expression matches of `Path` in route matches.
+- Supports exact and regular expression of `Headers` in route matches.
+- Supports `Method` in route matches.
+
+Supports of `TCPRoute` resource:
+- Supports multiple `BackendRefs` in `TCPRoute` resources for load balancing.
+
+Supports of `UDPRoute` resource:
+- Supports multiple `BackendRefs` in `UDPRoute` resources for load balancing.
+
+Supports of `TLSRoute` resource:
+- Supports `TLSRoute`.
```

Comment:
How about we make this a nested list instead?

```suggestion
- `Gateway` and `GatewayClass` resource:
  - Supports one `Gateway` and one `GatewayClass` to be handled by one KIC deployment.
  - Supports `allowedRoutes.namespaces` filters in listeners.
  - Supports referencing of cross-namespace `BackendRefs` if a ReferencePolicy permits them.

- `HTTPRoute` resource:
  - Supports multiple `BackendRefs` with a round-robin load-balancing strategy applied by default across the `Endpoints` or the `Services`, 
  and also now support weights of `BackendRefs` to enable more fine-tuning of the load-balancing between those backend services.
  - Supports exact, prefix, and regular expression matches of `Path` in route matches.
  - Supports exact and regular expression of `Headers` in route matches.
  - Supports `Method` in route matches.

- `TCPRoute` resource:
  - Supports multiple `BackendRefs` in `TCPRoute` resources for load balancing.

- `UDPRoute` resource:
  - Supports multiple `BackendRefs` in `UDPRoute` resources for load balancing.

- `TLSRoute` resource:
  - Supports `TLSRoute`.
```

Reply:
updated.